### ISSUE TITLE
Building dev container docker error

### ISSUE THREAD

**Issue by GitGud93** [Waztom/xchem-CAR#50]
Hi I got the following error while the dev container was busy setting up in VS code. At the top you will see several suggested options to resolve the issue. Help?
![image](https://user-images.githubusercontent.com/88319231/128148376-c23c83f8-c452-4608-88c7-edba7d9b56ce.png)

**Comment by Waztom** [comment]
OK, suspect this might be a DNS/network issue. Can you please try updating and upgrading your WSL by:
1. Inside your WSL terminal
2. `sudo apt update && sudo apt upgrade -y`
3. Alternatively, you can also test your connection by trying to ping a website -> `ping www.google.com `. You should see something like:
![image](https://user-images.githubusercontent.com/24798762/128154858-d21d096d-24f1-4870-9053-dc538e854c00.png)

**Comment by GitGud93** [comment]
So I think you were correct. I connected to a more stable/faster network and reloaded the window as per the top option (primarily so i could try a ping as it forced me to select an option before typing another command) and it suddenly worked just fine. I also saw that the line of code you suggested I run to update wsl was run by itself as part 6/6 of the process to creating the dev container. Thanks a bunch!
